CHANGELOG.md

```diff
@@ -1,6 +1,31 @@
 Magento Functional Testing Framework Changelog
 ================================================
 
+2.3.6
+-----
+### Enhancements
+* Maintainability
+    * A `-r` or `--remove` flag has introduced allowing for a command to clear out the contents of the `_generated` folder before generation. This flag has been added to the following commands:
```

Comment:
Should we mention it's part of bin/mftf commands option?

Reply:
Yes, I'll clarify this portion.